platform/android/export/export.cpp

```diff
@@ -36,13 +36,15 @@
 #include "editor/editor_settings.h"
 #include "editor/export/editor_export.h"
 
+#include <stdlib.h>
+
 void register_android_exporter_types() {
 	GDREGISTER_VIRTUAL_CLASS(EditorExportPlatformAndroid);
 }
 
 void register_android_exporter() {
 #ifndef ANDROID_ENABLED
-	EDITOR_DEF("export/android/android_sdk_path", "");
+	EDITOR_DEF("export/android/android_sdk_path", getenv("ANDROID_SDK_ROOT"));
```

Comment:
```suggestion
	EDITOR_DEF("export/android/android_sdk_path", OS::get_singleton()->get_environment("ANDROID_SDK_ROOT"));
```

Also, won't be useful on macOS (without https://github.com/godotengine/godot/pull/81266), since editor can't access shell environment (not a big deal, but should be mentioned if it's documented to auto pick it).

Reply:
I tested on a mac laptop and it seems to work as expected and picked up the env variable.

Reply:
It should work only when running the editor from terminal but not when running as a bundled app.